### PR TITLE
docs: add migration guide from bootc/netbox-chart to netbox-community/netbox-chart

### PR DESCRIPTION
**Pull request by davidusken**
Adding a migration guide on how to migrate from bootc/netbox-chart to netbox-community/netbox-chart

I can also add all the ArgoCD specific parts if this is of any interest

### CODE REVIEW

**Comment by RangerRick** [comment]
Thanks for this!

@LeoColomb and I were talking about this and are thinking we should break this up into 2 parts.
1. a general guideline on how to do an upgrade (dump the database, transition through minor versions)
2. taking these specific notes of yours and making them part of the "breaking changes" release notes for 5.0.0

For the former, I'm happy to modify this PR to turn it into a summary, if you'd like.
For the latter, we're tracking 5.0.0 stuff over in [this PR](https://github.com/netbox-community/netbox-chart/pull/286).

**Comment by RangerRick** [comment]
I've gone ahead and converted your `migrate.md` to a more generic wording, since the specifics got pulled into the 5.0 release PR. I'll merge this now.

Thanks again for doing the work of creating proper steps for migrating! It's really appreciated.

File: docs/migrate.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The first thing you should do is back up your PostgreSQL database.
+This way you can always go back to your previous install version if anything goes wrong.
+
+You can find your PostgreSQL pod by running `kubectl get pods -A | grep postgres` and then use `kubectl exec` to run `psql` or `pg_dump` from it.
+
+## Upgrade PostgreSQL If Necessary
+
+As of NetBox 3.6.x, NetBox requires PostgreSQL 12 or higher.
+It is recommmended that you upgrade to the latest supported PostgreSQL version.
```
Comment (LeoColomb) [inline docs/migrate.md:15]:
```suggestion
It is recommended that you upgrade to the latest supported PostgreSQL version.
```

**Review by LeoColomb — CHANGES_REQUESTED** [review]
One typo, otherwise looks perfect to me 👍

**Pull request merged**